community-operators/group-sync-operator/0.0.7/group-sync-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1103 @@
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        serviceAccountName: default
+      deployments:
+      - name: group-sync-operator-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: controller-manager
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                control-plane: controller-manager
+            spec:
+              containers:
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
+                resources: {}
+              - args:
+                - --metrics-addr=127.0.0.1:8080
+                - --enable-leader-election
+                command:
+                - /manager
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                image: quay.io/redhat-cop/group-sync-operator:v0.0.7
+                name: manager
+                resources:
+                  limits:
+                    cpu: 300m
+                    memory: 200Mi
+                  requests:
+                    cpu: 300m
+                    memory: 200Mi
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps/status
+          verbs:
+          - get
+          - update
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: default
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - group
+  - user
+  - security
+  - authorization
+  links:
+  - name: Group Sync Operator
+    url: https://github.com/redhat-cop/group-sync-operator
+  maintainers:
+  - email: ablock@redhat.com
+    name: Andrew Block
+  maturity: alpha
+  provider:
+    name: Red Hat Community of Practice
+    url: https://redhat-cop.github.io/
+  replaces: 0.0.6
```

Comment:
this should reference the CSV name to be replaced not the version

Reply:
Thanks for the help! Resolved the failures